Source/WebInspectorUI/UserInterface/Views/ResourceTreeElement.js

```diff
@@ -84,7 +84,19 @@ WI.ResourceTreeElement = class ResourceTreeElement extends WI.SourceCodeTreeElem
         if (comparisonResult !== 0)
             return comparisonResult;
 
-        // Compare by title when the subtitles are the same.
+        // Compare by title without file extensions when the subtitles are the same.
+        function filenameWithoutExtention (resourceTitle) {
+            let extentionPosition = resourceTitle.lastIndexOf("." + WI.fileExtensionForFilename(resourceTitle));
+            return resourceTitle.substring(0, extentionPosition);
+        }
+        
+        let titleA = filenameWithoutExtention(a.mainTitle);
+        let titleB = filenameWithoutExtention(b.mainTitle);
+        comparisonResult = titleA.extendedLocaleCompare(titleB);
+        if (comparisonResult !== 0)
+            return titleA.extendedLocaleCompare(titleB)
```

Comment:
Nit: can just `return comparisonResult`

Reply:
Addressed in 673a204c276c6ecb5d1c3dc8d0df42c54c1b0b67.